File: src/app/api/test/hello/route.ts
import { NextRequest, NextResponse } from 'next/server'

export async function GET(request: NextRequest) {
  return NextResponse.json({
    message: 'Hello from Next.js API route!',
    timestamp: new Date().toISOString(),
    status: 'working'
  })
}

export async function POST(request: NextRequest) {
  return NextResponse.json({
    message: 'POST request received from Next.js API route!',
    timestamp: new Date().toISOString(),
    status: 'working'
  })
} 

File: api/test/route.py
from http.server import BaseHTTPRequestHandler
import json

def handler(request):
    return {
        'statusCode': 200,
        'headers': {
            'Content-Type': 'application/json',
        },
        'body': json.dumps({
            'message': 'Python API is working!',
            'python_version': '3.x',
            'timestamp': '2025-07-01T12:00:00Z'
        })
    } 